tests/integration/test_issue_2441.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2024 Pex project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import
+
+import re
+
+import pytest
+
+from pex.pip.version import PipVersion
+from pex.typing import TYPE_CHECKING
+from testing import PY_VER
+from testing.cli import run_pex3
+
+if TYPE_CHECKING:
+    from typing import Any
+
+
+EXPECTED_ERROR_HEADER_LEAD_IN = (
+    "Found 7 invalid Requires-Dist metadata values in mlflow 1.27 metadata from "
+    "mlflow-1.27.0.dist-info/METADATA at"
+)
+
+EXPECTED_WHEEL = "mlflow-1.27.0-py3-none-any.whl"
+
+EXPECTED_ERROR_FOOTER = """\
+"scikit-learn (>=1.0.*) ; extra == 'pipelines'": .* suffix can only be used with `==` or `!=` operators
+    scikit-learn (>=1.0.*) ; extra == 'pipelines'
+                  ~~~~~~^
+"pyarrow (>=7.0.*) ; extra == 'pipelines'": .* suffix can only be used with `==` or `!=` operators
+    pyarrow (>=7.0.*) ; extra == 'pipelines'
+             ~~~~~~^
+"shap (>=0.40.*) ; extra == 'pipelines'": .* suffix can only be used with `==` or `!=` operators
+    shap (>=0.40.*) ; extra == 'pipelines'
+          ~~~~~~~^
+"pandas-profiling (>=3.1.*) ; extra == 'pipelines'": .* suffix can only be used with `==` or `!=` operators
+    pandas-profiling (>=3.1.*) ; extra == 'pipelines'
+                      ~~~~~~^
+"ipython (>=7.0.*) ; extra == 'pipelines'": .* suffix can only be used with `==` or `!=` operators
+    ipython (>=7.0.*) ; extra == 'pipelines'
+             ~~~~~~^
+"markdown (>=3.3.*) ; extra == 'pipelines'": .* suffix can only be used with `==` or `!=` operators
+    markdown (>=3.3.*) ; extra == 'pipelines'
+              ~~~~~~^
+"Jinja2 (>=3.0.*) ; extra == 'pipelines'": .* suffix can only be used with `==` or `!=` operators
+    Jinja2 (>=3.0.*) ; extra == 'pipelines'
+            ~~~~~~^
+"""
```

Comment:
@jasonwbarnett hopefully a bit better for the next person using --pip-version < 24.1. Thanks for initiating this.